qiime2/sdk/plugin_manager.py

```diff
@@ -142,34 +144,75 @@ def get_semantic_types(self):
 
         return types
 
+    class FormatFilters(enum.Flag):
```

Comment:
Maybe the name of the this class should be `GetFormatFilters`, or something along those lines. Any thoughts?